.github/workflows/release.yml

```diff
@@ -124,27 +155,27 @@ jobs:
   # NOTE: Watch out for the depth limit of 4 nested workflow_calls.
   # publish_layer -> publish_v2_layer -> reusable_deploy_v2_layer_stack -> reusable_update_v2_layer_arn_docs
   publish_layer:
-    needs: release
+    needs: [build, release]
```

Comment:
Maybe it's more correct to have it only depending on release (which already depends on build

```suggestion
    needs: [release]
```)

Reply:
nooo, it fails because of the `{{ needs.build.outputs.RELEASE_VERSION }}`, since the `build` object will be empty. Unless you want me to declare RELEASE_VERSION in the `release` job so it cascades down?

Reply:
ahahah I knew there was a reason :P It's good as it is then!